analytics/location_heatmaps/geo_utils.py

```diff
@@ -527,9 +521,46 @@ def compute_conf_intervals(sum_vector: np.ndarray, level=95):
   return conf_intervals, conf_interval_weighted
 
 
-def make_step(samples, eps, threshold, partial,
+def create_confidence_interval_condition(last_result, prefix, count, split_threshold):
```

Comment:
we're ultimately just returning a boolean here, so maybe evaluate_confidence_interval_condition instead? the current name makes me think that we're returning some kind of predicate function

Reply:
updated and added a docstring